cl/lib/pghistory.py

```diff
@@ -0,0 +1,79 @@
+import pgtrigger
+from pghistory import trigger
+from pghistory.core import DatabaseTracker, _get_name_from_label
+
+
+class CustomSnapshot(DatabaseTracker):
```

Comment:
Perhaps:

```suggestion
class AfterUpdateOrDeleteSnapshot(DatabaseTracker):
```

Calling something "custom" doesn't tell the next person much about why it exists. :)

Reply:
@mlissner Yes, the Snapshot class works for inserts and updates, we just talked about the behavior when we do an update, we don't have triggers for delete action, i can add them, i think we should also store the old value when we do a deletion, right?

Reply:
Yep, exactly.